src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Removed support for `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable. The
+  library will now emit
+  [stable](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0/docs/http)
+  semantic conventions by default.
```

Comment:
Rephrase this and remove "by default" to something like "The library will now emit only the stable semantic conventions".

Reply:
done